spec/Section 5 -- Validation.md

```diff
@@ -198,6 +198,67 @@ query getName {
 }
 ```
 
+### Subscription Operation Definitions
+
+#### Single root field
+
+**Formal Specification**
+
+  * For each subscription operation definition {subscription} in the document
+  * Let {rootFields} be the top level selection set on {subscription}.
+    * {rootFields} must be a set of one.
+
+**Explanatory Text**
+
+Subscription operations must have exactly one root field.
```

Comment:
Nullability may mean two different things in this context - this is a great point we should address.

One thing it may mean is that a subscription may not exist given some inputs in a way that isn't considered an error. I think this interpretation is both not what you were referring to, and also probably confusing to think about. The schema talks about the type of the *payload result* - so we're talking about the types of responses. We should probably make this point in the spec to clarify.

Secondly the nullability of the responses. This is one of the concerns with multi-field subscriptions to address later. For example, should it be legal to have a subscription field `streamThings: String?` where it is legal for any payload in the event sequence to in fact be `null`? I don't see a compelling reason to explicitly disallow this - though it is an edge case.

I think handling the payloads of multi-field subscriptions will need to account for this